src/docs/get-started/codelab-web.md

```diff
@@ -765,6 +766,10 @@ class _AnimatedProgressIndicatorState extends State<AnimatedProgressIndicator>
   }
 }
 ```
+
+The [didUpdateWidget][] function is called to 
```

Comment:
Add code font inside the brackets around `didUpdateWidget`, both here and when defining the URL below. In fact, can you change the sentence to:  "The [`didUpdateWidget`][] function updates the `AnimatedProgressIndicatorState` whenever `AnimatedProgressIndicator` changes.